### PR TITLE
F2F-388: Setup Infrastructure and permissions to consumer TxMA events

### PR DESCRIPTION
**Pull request by madankaruppiah**
IPVreturn services offer service to produce and consume events. To consume events, provide necessary permissions to lambda function to access the TxMA SQS queue and receive event. 
Added policies to receive event from TxMA queue to allow post event function to receive message.
Subscribe TxMA SQS to receive message form the cross account queue
 
- [F2F-388](https://govukverify.atlassian.net/browse/F2F-388)

[F2F-388]: https://govukverify.atlassian.net/browse/F2F-388?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: deploy/template.yaml [hunk 1]
```diff
@@ -678,6 +721,21 @@ Resources:
...
+      MessageRetentionPeriod:
+        !FindInMap [
+          EnvironmentVariables,
+          !Ref Environment,
+          MESSAGERETENTIONPERIODDAYS,
+        ]
+      VisibilityTimeout: 60
+      # KmsMasterKeyId: !Ref TxMAKeyAlias
+      # KmsMasterKeyAlias: "alias/aws/sqs"
+
```
Comment (darrendunford) [inline deploy/template.yaml:737]:
```suggestion
```

File: deploy/template.yaml [hunk 2]
```diff
@@ -60,14 +60,22 @@ Mappings:
   PlatformConfiguration:
     dev:
       CSLSEGRESS: arn:aws:logs:eu-west-2:885513274347:destination:csls_cw_logs_destination_prodpython
+      TxMASQS: arn:aws:sqs:eu-west-2:178023842775:self-staging-EC-SQS-Output-Queue-return
+      TxMAKey: arn:aws:kms:eu-west-2:178023842775:key/aba0d7ef-779a-4f6c-9b50-ec91f6cb8b5c
```
Comment (darrendunford) [inline deploy/template.yaml:64]:
```suggestion
```
not required environment is mocked

**Review by darrendunford — APPROVED** [review]

File: deploy/template.yaml (hunk 2 above)
Comment (AyeshaKhanGDS) [reply deploy/template.yaml]: removed

File: deploy/template.yaml (hunk 1 above)
Comment (AyeshaKhanGDS) [reply deploy/template.yaml]: removed

**Review by darrendunford — APPROVED** [review]

**Pull request merged**